### PR TITLE
allow for variable overrides via globals script

### PR DESCRIPTION
**Pull request by maxgrossman**
add a `globals` file that can be use to override external api's locations with if they differ from default.
doing overrides in this separate file allows for flexibility to make changes after an rpm is installed by just making changes to a single file.

For example if my the path to my translation server is different then you'd update the globals file to be.

```js
var tm4ApiUrl,
    translationServerPort = 'somewhere_else',
    mergeServerPort;
```

### CODE REVIEW

File: dist/globals [hunk 1]
```diff
@@ -0,0 +1,3 @@
+var tm4ApiUrl,
+    translationServerPort,
+    mergeServerPort;
```
Comment (brianhatchl) [inline dist/globals]: any reason not to name this one `globals.js`?

Comment (maxgrossman) [reply dist/globals]: this npm script removes all dist/*.js files and i was confused when i kept deleting `globals.js` unknowingly
https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/package.json#L17

Comment (maxgrossman) [reply dist/globals]: i'm all for a better solution if we have one.

Comment (brianhatchl) [reply dist/globals]: maybe change the rm to `dist/*.min.js`?

Comment (maxgrossman) [reply dist/globals]: 2️⃣ ⌚ 🔙 2️⃣ 🔙

**Review by brianhatchl — APPROVED** [review]
:alarm_clock: up & :lock: it in

**Pull request merged**